projects/emissary-ingress/emissary/docker/linux/emissary-image/Dockerfile

```diff
@@ -0,0 +1,59 @@
+ARG BASE_IMAGE 
+ARG BUILDER_IMAGE
+ARG APPMESH_VERSION
+ARG TARGETARCH
+ARG TARGETOS
+FROM public.ecr.aws/appmesh/aws-appmesh-envoy:$APPMESH_VERSION as envoy
+
+FROM $BUILDER_IMAGE as emissary-builder
+
+ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin:/buildroot/bin:/newroot/buildroot
+
+COPY emissary/python _output/bin/emissary/linux-amd64 emissary/manifests/emissary/emissary-crds.yaml.in /newroot/buildroot
+COPY --from=envoy /usr/bin/envoy /THIRD-PARTY-LICENSES.txt /
+
+WORKDIR /newroot/buildroot
+
+RUN install_rpm libcap htop && \
+    install_binary /usr/bin/curl /usr/bin/chgrp /usr/bin/chmod && \
+    time pip3 install --no-deps -e . && \
+    pip3 install pip-tools==6.3.1 && \
+    pip3 install PyYAML==5.4.1 && \
+    pip3 install orjson==3.6.6 && \
+    mkdir -p /newroot/ambassador/sidecars /newroot//usr/lib/python3.9/site-packages /newroot/usr/local/bin /ambassador /buildroot && \
+    mv /envoy /newroot/usr/local/bin/envoy && \
+    mv /THIRD-PARTY-LICENSES.txt /newroot/ENVOY_LICENSES.txt && \
+    cp -r /usr/lib/python3.9/site-packages /newroot/usr/lib/python3.9 && \
+    mkdir -p /newroot/buildroot/manifests/emissary && \
+    mv emissary-crds.yaml.in /newroot/buildroot/manifests/emissary/emissary-crds.yaml && \
+    ln -sf /newroot/buildroot/post_update.py /ambassador/post_update.py && \
```

Comment:
These symlinks with `/newroot` in them do not seem right to me since what i believe it is saying is

`/ambassador/post_update.py` -> `/newroot/buildroot/post_update.py` and on the final image the /newroot directory will nto exists, it will just be `/`.  The ones from on 32-35,, pointing to `/buildroot` seem reasonable.

Reply:
Fixed this too!